src/service/mobile_config.proto

```diff
@@ -43,6 +43,13 @@ message cbrs_radio_deployment_info {
 }
 
 message gateway_metadata {
+  option deprecated = true;
```

Comment:
i've only ever tried to deprecate fields with this notation, not entire messages but in those cases rust/prost seems to be unable to handle the message being deprecated and omitted without throwing missing field warnings. hopefully it's easier when the whole message is flagged

Reply:
Honestly, I’ve never thought about how Rust/Prost handles this deprecation, but I didn’t see any related warnings while working on this PR.